### PR TITLE
added elyra pipeline automation

### LINKED ISSUE #13: Automate notebooks for Demo #1 using Elyra pipelines
Demo 1 has notebooks which ingest raw data from S3 and update them as tables on Trino.

We want to automate these notebooks from #4 by setting up Elyra pipelines which can run on Kubeflow Pipelines.

Acceptance Criteria:
- [ ] Make necessary modifications in notebooks from #4 to make them suitable to run using Elyra pipelines.
- [ ] Create elyra pipeline config, add the configuration to the repo and set up a demo pipeline.

### PR DESCRIPTION
**Pull request by oindrillac**
resolves #13 

A demo1 pipeline was setup and notebooks were successfully run in an elyra pipeline in automation.

image

### CODE REVIEW

File: notebooks/demo1.pipeline [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        "ui_data": {
+          "comments": []
+        },
+        "version": 3
+      },
+      "runtime_ref": ""
+    }
+  ],
+  "schemas": []
+}
```
Comment (MichaelClifford) [inline notebooks/demo1.pipeline:149]: can you add a new line at end of file?

**Review by MichaelClifford — CHANGES_REQUESTED** [review]
very small nit. otherwise :+1:

File: notebooks/demo1.pipeline (hunk 1 above)
Comment (oindrillac) [reply notebooks/demo1.pipeline:149]: sure

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm

**Review by MichaelClifford — APPROVED** [review]

**Pull request merged**